scripts/test-k8s-dataflow.sh

```diff
@@ -793,38 +793,18 @@ verify_recommendations() {
         echo_info "Port 32061 may have stopped working since helm installation"
     fi
 
-    # Test API status endpoint (public endpoint - no auth needed) with retries
-    echo_info "Testing ROS-OCP API status..."
-    echo_info "Status URL: $status_url"
-
-    local max_retries=5
-    local retry_delay=10
-    local status_http_code="000"
-
-    for i in $(seq 1 $max_retries); do
-        echo_info "Attempt $i/$max_retries to reach rosocp-api..."
-        local status_response=$(curl -s -w "%{http_code}" --connect-timeout 5 --max-time 15 -o /tmp/status_response.json \
-            -H "Host: localhost" \
-            "$status_url" 2>/dev/null || echo "000")
-
-        status_http_code="${status_response: -3}"
-
-        if [ "$status_http_code" = "200" ]; then
-            echo_success "ROS-OCP API status endpoint is accessible"
-            if [ -f /tmp/status_response.json ]; then
-                echo_info "Status response: $(cat /tmp/status_response.json)"
-                rm -f /tmp/status_response.json
-            fi
-            break
-        else
-            echo_warning "Attempt $i failed (HTTP $status_http_code), retrying in ${retry_delay}s..."
-            if [ $i -lt $max_retries ]; then
-                sleep $retry_delay
-            fi
-        fi
-    done
-
-    if [ "$status_http_code" != "200" ]; then
+    # NOTE: Skipping /status endpoint check due to known Ingress routing issue in ros-helm-chart
+    # The greedy path rule (path: /, pathType: Prefix) causes /status to route incorrectly.
+    # This is a known issue that affects Kubernetes deployments with nginx ingress controller.
+    # OpenShift deployments (using Routes) are not affected.
+    # TODO: Fix ingress.yaml path ordering and add explicit /status route
+    echo_info "Skipping /status endpoint check (known Ingress routing issue in ros-helm-chart)"
+    echo_info "Direct pod access to /status works, but ingress routing fails"
+    echo_info "This will be fixed in a future PR by adding explicit /status route to ingress.yaml"
```

Comment:
do not forget to open a  follow-up PR :)

Reply:
Ops, that's no longer necessary. I found out that my problem were the network policies blocking the access. I will remove this